app/javascript/articles/components/SaveButton.jsx

```diff
@@ -8,20 +8,23 @@ export class SaveButton extends Component {
 
     const { isBookmarked } = props;
 
-    this.state = { buttonText: isBookmarked ? 'Saved' : 'Save' };
+    this.state = {
+      buttonText: isBookmarked ? 'Saved' : 'Save',
+      hovering: false,
+    };
   }
 
   render() {
-    const { buttonText } = this.state;
+    //const { hovering } = this.state.hovering;
```

Comment:
Ah just noticed this. Did you mean to use or remove this variable?

Reply:
Yeah i think they wanna get rid of the hovering state property. Should I just commit all these changes? lol

Reply:
I think Nick's suggestions make sense!